plugins/Files/js/components/deletedialog.js

```diff
@@ -13,7 +13,8 @@ const DeleteDialog = ({files, actions}) => {
 			<div className="delete-dialog">
 				<h3> Confirm Deletion </h3>
 				<div className="delete-text">
-					Are you sure you want to delete {files.size} {files.size === 1 ? ' file' : ' files'}
+					Are you sure you want to delete {files.size}{' '}
+					{files.size === 1 ? ' file' : ' files'}
```

Comment:
the `{' '}` looks wrong to me, given that the following strings are also prefixed by a space.

Reply:
{' '} represents a space. It's somewhat verbose but generally clearer to read because we're writing a mixture of JS and HTML in these files.

Reply:
Sure, but in a lot of these cases we don't really care about the space. I think we often did stuff like:
```html
<h1> Some text here </h1>
```
to separate the content from the tags, but the formatter thinks that we want to render that extra space -- we don't.